### DISCUSSION TITLE
Where can i find a sample dictionary file for populating tables

### DISCUSSION THREAD

**Discussion by drnkwati** [pawelsalawa/sqlitestudio#3420] [Q&A]
Hi
I need a demo for dictionary file used for populating table field when setting configure.
thanks

**Comment by pawelsalawa** [comment]
Dictionary file is just any text file. From that file every word will be used as another populating value. You can change the behavior to take every line as a value, instead of every word.

You can also define whether values from that file should be populated into the table using the same order as they go in the file, or maybe you would like to make the order random.
